compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/CallableReferenceLowering.kt

```diff
@@ -72,24 +72,17 @@ internal class CallableReferenceLowering(private val context: JvmBackendContext)
         expression.transformChildrenVoid(this)
 
         if (expression.valueArgumentsCount < FunctionInvokeDescriptor.BIG_ARITY ||
-            !expression.symbol.owner.parentAsClass.defaultType.isFunctionOrKFunction())
-            return expression
-
-        return IrCallImpl(
-            expression.startOffset, expression.endOffset,
-            expression.type, functionNInvokeFun.symbol, functionNInvokeFun.descriptor,
-            1, expression.origin
-        ).apply {
-            putTypeArgument(0, expression.type)
-            dispatchReceiver = expression.dispatchReceiver
-            extensionReceiver = expression.extensionReceiver
```

Comment:
Because the function in question doesn't actually have type parameters or an extension receiver parameter. Previously the IrCall was just built manually, with an incorrect number of type arguments.